backend/de.metas.salescandidate.base/src/main/java/de/metas/ordercandidate/api/OLCandRegistry.java

```diff
@@ -44,6 +45,9 @@
 @Component
 public class OLCandRegistry
```

Comment:
wdyt about renaming it to OLCandSPIRegistry? because it contains all those interfaces..